src/Avalonia.Controls/Button.cs

```diff
@@ -207,6 +208,9 @@ protected override void OnDetachedFromVisualTree(VisualTreeAttachmentEventArgs e
 
         protected override void OnAttachedToLogicalTree(LogicalTreeAttachmentEventArgs e)
         {
+            if (_hotkey != null)
+                // Control attached again, set Hotkey to create a hotkey manager for this control
```

Comment:
use braces and move comment to next to the if ()

Reply:
Done